src/blocks/BulletPointBlock/BulletPointBlock.tsx

```diff
@@ -0,0 +1,217 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  ContentWrapper,
+  getMinimalColorStyles,
+  LAPTOP_BP_UP,
+  MOBILE_BP_DOWN,
+  MOBILE_BP_UP,
+  SectionWrapper,
+  TABLET_BP_UP,
+} from 'components/blockHelpers'
+import { DeferredImage } from 'components/DeferredImage'
+import { getStoryblokImage, Image } from 'utils/storyblok'
+import {
+  BrandPivotBaseBlockProps,
+  MarkdownHtmlComponent,
+  MinimalColorComponent,
+} from 'blocks/BaseBlockProps'
+import { FontSizes, Heading } from 'components/Heading/Heading'
+
+const BulletPointSectionWrapper = styled(SectionWrapper)`
+  overflow-x: hidden;
+`
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+
+  ${MOBILE_BP_DOWN} {
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+
+  ${MOBILE_BP_UP} {
+    margin-left: -1.5rem;
```

Comment:
Why do we need the minus margins? Maybe it's just me but I always get a little nervous by minus margins, I feel like they can mess things up 😅

Reply:
Ah, I moved the existing component so most of the component is old code. This is a solution we use to make the column layout with `flexbox` to get tha gap between items. Each item has a `margin-left` and then you offset the wrapper with the same negative value to center it.

Reply:
Ah OK, I see! Well I'm probably just too nervous generally 😅

Reply:
Great that you asked! 👌